sql/sql_type.h

```diff
@@ -979,6 +978,62 @@ class Information_schema_character_attributes
 };
 
 
+enum vers_kind_t
+{
+  VERS_UNDEFINED= 0,
```

Comment:
What do you still need VERS_UNDEFINED for?

Reply:
It means table is unversioned.